### PR TITLE
homepage + contact form

### PR DESCRIPTION
**Pull request by actlikewill**

### CODE REVIEW

File: peachjam/templates/peachjam/_contact_form.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{% load i18n %}
+<section class="row ">
+  <div class="my-4 col-lg-6 card">
```
Comment (longhotsummer) [inline peachjam/templates/peachjam/_contact_form.html:3]: I suggest dropping the card; if you do want the card, don't put it on the col.

Also add: `col-12`

File: peachjam/views/about.py [hunk 2]
```diff
@@ -1,6 +1,17 @@
...
+
+class AboutPageView(FormView):
+    form_class = ContactUsForm
     template_name = "peachjam/about.html"
     navbar_link = "about"
+    success_url = reverse_lazy("about")
+
+    def form_valid(self, form):
+        form.send_email()
+        messages.success(self.request, "Your message has been sent. Thank you!")
```
Comment (longhotsummer) [inline peachjam/views/about.py:16]: translateable

File: peachjam/templates/peachjam/emails/contact_us_email.txt [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{{ APP_NAME }} Contact Form Message
+A {{ APP_NAME }} user has sent a message through the contact form:
+Message:
+{{ message }}
+Sent by: {{ email_address }}
```
Comment (longhotsummer) [inline peachjam/templates/peachjam/emails/contact_us_email.txt:5]: add `{{ name }}`

File: peachjam/forms.py [hunk 4]
```diff
@@ -367,6 +367,45 @@ def send_email(self):
         )
 
 
+class ContactUsForm(forms.Form):
+    name = forms.CharField(max_length=255, required=True)
```
Comment (longhotsummer) [inline peachjam/forms.py:371]: add a honeypot field

Comment (actlikewill) [reply peachjam/forms.py:374]: added

File: peachjam/views/about.py (hunk 2 above)
Comment (longhotsummer) [reply peachjam/views/about.py:16]: The success message must be translateable.

**Comment by longhotsummer** [comment]
Looks great - just the success message to be marked as translateable.

File: peachjam/views/about.py (hunk 2 above)
Comment (actlikewill) [reply peachjam/views/about.py]: right, done

**Review by longhotsummer — APPROVED** [review]

File: peachjam/fixtures/documents/sample_documents.json [hunk 5]
```diff
@@ -29,6 +29,7 @@
         "pk": 2,
         "fields": {
             "name": "High Court",
+            "slug": "high-court",
```
Comment (actlikewill) [inline peachjam/fixtures/documents/sample_documents.json:32]: @longhotsummer added this to the test data so the kl tests can pass

**Pull request merged**